### PR TITLE
docs(readMe): jira ticket Test

### PR DESCRIPTION
**Pull request by jparangdev**
- 지라와 연동을 위한 테스트

[CST-14]

[CST-14]: https://32devs.atlassian.net/browse/CST-14?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,3 +4,4 @@ Pages for css learning
 
 - GitHub : https://github.com/32devs/css-selector-trainer
 - FrontEnd : React + Typescript
+- backEnd : spring boot + jpa
```
Comment (hasuikhs) [inline README.md]: this is test

Comment (sinbpark) [inline README.md]: 남깁니다

**Review by hasuikhs — COMMENTED** [review]
leave review

File: README.md (hunk 1 above)
Comment (jparangdev) [reply README.md]: ok

**Review by hasuikhs — COMMENTED** [review]
ok good

File: README.md [hunk 2]
```diff
@@ -4,3 +4,4 @@ Pages for css learning
 
 - GitHub : https://github.com/32devs/css-selector-trainer
 - FrontEnd : React + Typescript
+- BackEnd : Springboot + JPA
```
Comment (sinbpark) [inline README.md:7]: 가랏!

**Comment by sinbpark** [comment]
LGTM

**Review by sinbpark — COMMENTED** [review]
LGTM

**Review by hasuikhs — APPROVED** [review]
LGTM

**Review by sinbpark — APPROVED** [review]
submit review

**Pull request merged**